pkg/component/watch.go

```diff
@@ -134,27 +134,32 @@ func WatchAndPush(client *occlient.Client, componentName string, applicationName
 		for {
 			select {
 			case event := <-watcher.Events:
+				isIgnoreEvent := false
 				changeLock.Lock()
 				glog.V(4).Infof("filesystem watch event: %s", event)
 
 				if !(event.Op&fsnotify.Remove == fsnotify.Remove || event.Op&fsnotify.Rename == fsnotify.Rename) {
 					stat, err := os.Lstat(event.Name)
 					if err != nil {
-						glog.Errorf("Failed getting details of the changed file %s", event.Name)
-						watchError = errors.Wrap(err, "unable to watch changes")
+						// Some of the editors generate temporary buffer files during update to the file and deletes it soon after exiting from the editor
```

Comment:
can you please mention the editors name as well for which you faced the problem

Reply:
Ack Done